Jenkinsfile

```diff
@@ -235,6 +256,20 @@ ansiColor('xterm') {
               }
             }
 
+            // Define test URLs for the integration (test) environment.
```

Comment:
Would it be worthwhile to get these from the recently updated `.env.default` file instead?

Reply:
Eventually it will just pull from updated credentials on Jenkins.